src/scipp/plotting/figure.py

```diff
@@ -31,13 +31,13 @@ def __init__(self,
         self.own_axes = True
         self.toolbar = None
         self.padding = padding
+        cfg = config['plot']
         if self.ax is None:
             if figsize is None:
-                figsize = (config.plot.width / config.plot.dpi,
-                           config.plot.height / config.plot.dpi)
-            self.fig, self.ax = plt.subplots(1, 1, figsize=figsize, dpi=config.plot.dpi)
+                figsize = (cfg['width'] / cfg['dpi'], cfg['height'] / cfg['dpi'])
+            self.fig, self.ax = plt.subplots(1, 1, figsize=figsize, dpi=cfg['dpi'])
             if self.padding is None:
-                self.padding = config.plot.padding
+                self.padding = cfg['bounding_box']
```

Comment:
Does this now mean that the plotting function accepts an argument named `padding`, but the option in the config is called `bounding_box`?
-> change everything to `bounding_box`?

Reply:
> -> change everything to bounding_box?

Will do.